examples/counter/components/Counter.js

```diff
@@ -23,3 +16,12 @@ export default class Counter extends Component {
     );
   }
 }
+
+Counter.propTypes = {
+  increment: PropTypes.func.isRequired,
+  incrementIfOdd: PropTypes.func.isRequired,
+  decrement: PropTypes.func.isRequired,
+  counter: PropTypes.number.isRequired
+};
+
+export default Counter;
```

Comment:
I thought it works fine to `export default class Foo` at the top and still add propTypes after the class declaration.

(Forgive me if I'm wrong, I can't test it out right now.)